docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -3224,6 +3393,38 @@ Instruct the driver to remove the volume (`name`).
 -   **409** - volume is in use and cannot be removed
 -   **500** - server error
 
+### Prune unused volumes
+
+`POST /volumes/prune`
+
+Delete unused volumes
+
+**Example request**:
+
+    POST /volumes/prune HTTP/1.1
+    Content-Type: application/json
+
+    {
+    }
+
+**Example response**:
+
+    HTTP/1.1 200 OK
+    Content-Type: application/json
+
+	{
+	    "VolumesDeleted": [
+	        "my-volume"
```

Comment:
wdym? It'd say it's the same format 

There's no notion of "untagged" and "Deleted" for volumes, so it just return the name/id of the volume instead.